crates/bevy_reflect/src/lib.rs

```diff
@@ -1807,10 +1793,10 @@ bevy_reflect::tests::should_reflect_debug::Test {
     map: {
         123: 1.23,
     },
-    a_struct: bevy_reflect::tests::should_reflect_debug::SomeStruct {
+    a_struct: bevy_reflect::tests::SomeStruct {
```

Comment:
Hm, does `TypePath` mention this quirk (due to `module_path!`)? If not, that might be worth adding as part of this PR so users who are defining structs inside functions aren't wholly caught off guard.